system_setup/cmd/tui.py

```diff
@@ -48,28 +48,16 @@ def make_client_args_parser():
     parser.add_argument('--dry-run', action='store_true',
                         dest='dry_run',
                         help='menu-only, do not call installer function')
-    # TODO WSL: remove any uneeded arguments
     parser.add_argument('--socket')
     parser.add_argument('--serial', action='store_true',
                         dest='run_on_serial',
                         help='Run the installer over serial console.')
-    parser.add_argument('--ssh', action='store_true',
-                        dest='ssh',
-                        help='Print ssh login details')
     parser.add_argument('--ascii', action='store_true',
                         dest='ascii',
                         help='Run the installer in ascii mode.')
     parser.add_argument('--unicode', action='store_false',
```

Comment:
Same here. (and hilight the benefit/where ascii mode would be required for WSL users)